yk_face/face.py

```diff
@@ -83,3 +84,25 @@ def identify(face_template: str, group_id: str, minimum_score: float = -1.0,
                                        minimum_score=minimum_score,
                                        gallery_id=group_id).to_dict()
     return request('POST', url, json=identify_request)
+
+
+def verify_images(first_image, second_image) -> float:
+    """
+        Verifies if the face detected on the first image belongs to the
+        face detected on the second image.
+    :param first_image:
+        A base64 string or a file path or a file-like object representing an image.
+    :param second_image:
+        A base64 string or a file path or a file-like object representing an image.
+    :return:
+        Matching Score.
+    """
+    first_face = process(first_image)
+    second_face = process(second_image)
+
+    if not first_face or not second_face:
```

Comment:
It is better to check each one individually and specify which image failed in the Exception text.